core/src/user.rs

```diff
@@ -44,30 +44,9 @@ where
         }
     }
 
-    #[tracing::instrument(skip(self), err(level = tracing::Level::ERROR), ret(level = tracing::Level::TRACE))]
-    pub fn deposit_tx(
```

Comment:
Can we also re-add this, and change the tests to use this maybe?

Reply:
2 of the return values were known by the caller and not modified inside of the function. So, they are removed.